src/About/About.tsx

```diff
@@ -1,49 +1,54 @@
-import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
-import { translate } from 'react-i18next';
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { translate } from "react-i18next";
 
 import {
 	Button,
 	Typography,
 	Card,
 	CardContent,
-	CardMedia,
-} from '@material-ui/core';
-import Carousel from 'nuka-carousel';
+	CardMedia
+} from "@material-ui/core";
+import Carousel from "nuka-carousel";
 import {
 	NavigateNext as NavigateNextIcon,
-	NavigateBefore as NavigateBeforeIcon,
-} from '@material-ui/icons';
+	NavigateBefore as NavigateBeforeIcon
+} from "@material-ui/icons";
 
-import { getResponsiveImageUrl } from '../ResponsiveImage/ResponsiveImage';
-import Container from '../Container/Container';
-import OurStack from '../OurStack/OurStack';
-import Team from './Team';
-import css from './About.module.css';
+import { getResponsiveImageUrl } from "../ResponsiveImage/ResponsiveImage";
+import Container from "../Container/Container";
+import OurStack from "../OurStack/OurStack";
+import Team from "./Team.json";
+import css from "./About.module.css";
 
-function shuffleArray(array) {
+type AboutProps = any;
+
+function shuffleArray(array: number[]) {
 	for (let i = array.length - 1; i > 0; i--) {
-		let j = Math.floor(Math.random() * (i + 1));
+		const j = Math.floor(Math.random() * (i + 1));
 		[array[i], array[j]] = [array[j], array[i]];
 	}
 	return array;
 }
 
 const cardWidth = 300;
 
-@translate(['about'], { wait: true })
-class About extends Component {
-	render() {
+// Fixme: this is a workaround for using the material ui button
```

Comment:
Yep!

Reply:
In fact, this is the only long term issue I see. Maybe we should just create a `ToButton` component that does exactly this and use that instead of the `CustomButton` that is added to several files now?

Does this error mean that the type provided by MaterialUI for `Button` is wrong? Because in that case we can do a PR as a long term solution.